mlir/lib/Dialect/SPIRV/IR/SPIRVTypes.cpp

```diff
@@ -151,6 +155,14 @@ void CompositeType::getExtensions(
         return llvm::cast<ScalarType>(type.getElementType())
             .getExtensions(extensions, storage);
       })
+      .Case<TensorArmType>([&](TensorArmType type) {
+        static const Extension exts[] = {Extension::SPV_ARM_tensors};
+        ArrayRef<Extension> ref(exts, std::size(exts));
```

Comment:
You don't need to create an array and the array ref here, array ref has a constructor that takes one element